idea/testData/joinLines/stringTemplate/simple.kt.after

```diff
@@ -0,0 +1,3 @@
+fun test1() {
+    val s = <caret>"foobar"
```

Comment:
should the caret maybe placed to the same place, i.e.
```kotlin
"foo<caret>bar"
```

Reply:
Yes, the caret needs to be in the same place.

Reply:
Fixed